pkg/api/schema/schema.proto

```diff
@@ -376,6 +377,7 @@ message ZScanRequest {
 	Score maxScore = 9;
 	uint64 sinceTx = 10;
 	bool  noWait = 11;
+	uint64 offset = 12;
```

Comment:
Would be good to have some small comment here (that will propagate to generated GRPC and SWAGGER code)